images/linux/scripts/installers/vcpkg.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+################################################################################
+##  File:  vcpkg.sh
+##  Team:  CI-Platform
+##  Desc:  Installs VCPKG
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+git clone --depth=1 https://github.com/Microsoft/vcpkg
```

Comment:
Please double check target install directory, does it align with other software? - for example 
LEIN_BIN=/usr/local/bin/lein